pkg/config/config.go

```diff
@@ -12,6 +12,7 @@ type Config struct {
 	Dir                  string
 	DisableVersionString bool `mapstructure:"disable-version-string"`
 	DryRun               bool `mapstructure:"dry-run"`
+	Exported             bool
```

Comment:
Just for consistency let's add the explicit mapstructure tag here (really I would like all of them to have it).

Reply:
fixed